common/api/core-common.api.md

```diff
@@ -331,6 +331,75 @@ export namespace AreaPattern {
     }
 }
 
+// @public
+export namespace AtmosphericScattering {

```

Comment:
Just call this `Atmosphere`.

Reply:
@pmconne All checks have passed, but there are some unresolved comments you left from a while back. Could you revisit those when you have the time?